filter/fes-2.0.2/src/main/java/org/orbisgis/orbisdata/filter/fes_2_0_2/FesToSql.java

```diff
@@ -109,4 +138,256 @@ public static StringBuilder XmlToSql(Object objectFromFilterXml) {
         }
         return returnSQL;
     }
+
+
+//------------------------------------------------Operator Comparison-------------------------------------------------
+
+    /**
+     * Method separate the comparison object for each comparison operator
+     * @param comparisonElement
+     * @return
```

Comment:
Do not forget to add description to `@param` and `@return`.
(same thing for the other headers)